pkg/monitoring/rules/rules.go

```diff
@@ -0,0 +1,180 @@
+package rules
+
+import (
+	"fmt"
+
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
nit: newline

Reply:
No, I usually split the `import` block into 3 parts:
- standard library
- imports from other modules defined in `go.mod`
- imports from this project